lib/mocha.js

```diff
@@ -249,7 +252,7 @@ Mocha.prototype.reporter = function(reporter, reporterOptions) {
       );
```

Comment:
At lines (L226, L231), be nice to add the reciprocal processing...
```js
 if (err.code !== 'MODULE_NOT_FOUND' ||
     err.message.indexOf('Cannot find module') !== -1) {
```

Reply:
In Node.js v6 you can use `String.prototype.includes`; `err.message.includes('Cannot find module')`.

Reply:
You fixed the first one, but skipped the second.